src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -205,13 +205,17 @@ public static IndexReader getReader(String path) throws IOException {
 
   public static Map<String, Long> getTermCounts(IndexReader reader, String termStr) throws IOException, ParseException {
     EnglishAnalyzer ea = new EnglishAnalyzer(CharArraySet.EMPTY_SET);
-    QueryParser qp = new QueryParser(IndexArgs.CONTENTS, ea);
+    return getTermCounts(reader, termStr, ea);
+  }
+
+  public static Map<String, Long> getTermCounts(IndexReader reader, String termStr, Analyzer analyzer) throws IOException, ParseException {
```

Comment:
https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/index/IndexReaderUtils.java#L316

I don't think this will work w/ pyjnius? Just rename method `getTermCountsWithAnalyzer`